draft-annevk-johannhof-httpbis-cookies.md

```diff
@@ -760,23 +760,44 @@ To **Remove Excess Cookies for a Host** given a host _host_:
 
 1. Sort _secureCookies_ by earliest last-access-time first.
 
+1. Let _excessHostCookies_ be an empty list of cookies.
+
 1. While _insecureCookies_'s size + _secureCookies_'s size is greater than the user agent's total cookies-per-host limit:
 
-    1. If _insecureCookies_ is not empty, remove the first item of _insecureCookies_ and
-    delete the corresponding cookie from the user agent's cookie store.
+    1. If _insecureCookies_ is not empty:
+        
+        1. Remove the first item of _insecureCookies_,
```

Comment:
I wonder if we can clarify this a bit more. Perhaps like this:

1. Let _cookie_ be _insecureCookies_[0].
2. Remove _cookie_ from _insecureCookies_.
3. Remove _cookie_ from the user agent's cookie store.
4. Append _cookie_ to _excessHostCookies_.

We might have to clarify above then that _insecureCookies_ holds references. WDYT?